### ISSUE TITLE
Updates to naming convention in the Database

### ISSUE THREAD

**Issue by GernotMaier** [gammasim/simtools#799]
(this is related to #797)

Rewrite database with simplified naming as follows:

Keyword updates
- File -> file
- Type -> type
- Value -> value
- unit -> units

New keywords

- site: South/North
- array_element
- version

New instrument naming:

- LSTN-01 and LSTN-design
- MSTN-design, LSTS-design, MSTS-design, SSTS-design, SCTS-design
- be prepared for calibration devices (e.g., ILLN-01)

Telescope naming:

- North-LST-1 --> LSTN-01
- North-LST-D234 --> LSTN-design
- North-MST-FlashCam-D --> removed
- North-MST-NectarCam-D --> MSTN-design
- North-MST-Structure-D --> MSTN-design
- South-LST-D --> LSTS-design
- South-MST-FlashCam-D --> MSTS-design
- South-MST-NectarCam-D --> removed
- South-MST-Structure-D --> MSTS-design
- South-SCT-D --> SCTS-design
- South-SST-1M-D --> removed
- South-SST-ASTRI-D --> removed
- South-SST-Camera-D --> SSTS-design
- South-SST-GCT-D --> removed
- South-SST-Structure-D --> SSTS-design

**Comment by orelgueta** [comment]
> * File -> file
> * Type -> type
> * Value -> value
> * unit -> units

We have more parameters which start with a capital letter (`Parameter`, `Applicable`, perhaps more). Do you want those to change as well to lower case?

BTW, `unit` is already `units` in the DB. Did you mean the other way around?


> * site: South/North
> * array_element
> * version

- What do you mean by site here? I thought the site will be in the telescope name. Do you want it explicit as well?
- version we already have, but with a capital letter, so I will change it.
- array_element would replace "Telescope" I guess?

Apologies if we discussed these questions already, but I want to make sure before I make changes.

**Comment by GernotMaier** [comment]
> > * File -> file
> > * Type -> type
> > * Value -> value
> > * unit -> units
> 
> We have more parameters which start with a capital letter (`Parameter`, `Applicable`, perhaps more). Do you want those to change as well to lower case?

I think we decided to change all for consistency - so yes, we should change all parameters.

> 
> BTW, `unit` is already `units` in the DB. Did you mean the other way around?

My mistake, we want to have `unit` to be consistent with astropy (where `.unit` gives you the unit of a value).

> 
> > * site: South/North
> > * array_element
> > * version
> 
>     * What do you mean by site here? I thought the site will be in the telescope name. Do you want it explicit as well?

At least for the site parameters we need to have an explicit site field. You are right that this is not needed for the telescope model and we can remove it here.

> 
>     * version we already have, but with a capital letter, so I will change it.
> 
>     * array_element would replace "Telescope" I guess?
> 

OK, agree!

> Apologies if we discussed these questions already, but I want to make sure before I make changes.

No worries, I get constantly confused by the changes.

**Comment by orelgueta** [comment]
> - array_element would replace "Telescope" I guess?

I actually wonder about this. Wouldn't the other array elements like the illuminators follow a different schema and might be in a different collection? If so, "telescope" should stay here as the name of the array element and in the other collection we can use "array_element". What do you think?

**Comment by GernotMaier** [comment]
This is fine with me.

Did we decide on dropping `site` in the telescope collection? We would still need it in the Site collection.

**Comment by GernotMaier** [comment]
(apologies for being indecisive)

Another option would be to keep it to be `instrument`. This would logically allow to drop the site field for all collections (telescope, calibration devices; for 'Site' the instrument would be 'CTAO-North', ...). We use 'INSTRUMENT' also in the CTAO Metadata context, so this would be consistent.

Should we do this?

**Comment by orelgueta** [comment]
I am not sure if consistency should be our guide. I think clarity is more important. In case of the telescopes, `instrument` would be OK (but less ideal than `telescope`). While `instrument: CTAO-North` would be nothing but confusing (also remember we decided the site should be `North`, not `CTAO-North`. 

Since the schema for site and telescope is anyway different, I don't see why we need to fight for consistency. I would prefer to keep `telescope` in the telescopes collection and if you want I can also add an explicit `site` field (earlier we decided not to, but it's not a big deal). For the site collection we will have a `site` field. And later we add another collection for the array elements.
